src/format.rs

```diff
@@ -88,6 +88,12 @@ impl VetVersion {
             })
         }
     }
+
+    /// Check if this VetVersion exactly matches the given semver version with
+    /// no git revision metadata.
+    pub fn equals_semver(&self, semver: &semver::Version) -> bool {
+        self.git_rev.is_none() && &self.semver == semver
+    }
```

Comment:
Could also add a `PartialEq<semver::Version>` impl.

Reply:
Yeah, I felt like making it explicit made sense here because so that folks don't accidentally compare the semver component from e.g. a cargo_metadata package instead of calling `vet_version()`, but that'd probably also be fine.